features/tailormade_auth/src/main/java/com/future/tailormade_auth/core/model/request/SignUpRequest.kt

```diff
@@ -0,0 +1,14 @@
+package com.future.tailormade_auth.core.model.request
+
+data class SignUpRequest(
```

Comment:
for mandatory field, please remove the default value
it will useful in code thee feature and unit testing